addons/road-generator/connectors/terrain3d_road_connector.gd

```diff
@@ -0,0 +1,220 @@
+@tool
+extends Node
+
+const RoadSegment = preload("res://addons/road-generator/nodes/road_segment.gd")
+
+# Terrain3D
+## Reference to the Terrain3D instance, to be flattened
+@export var terrain:Terrain3D:
+	set(value):
+		terrain = value
+		configure_road_update_signal()
+## Reference to the RoadManager instance, read only
+@export var road_manager:RoadManager:
+	set(value):
+		road_manager = value
+		configure_road_update_signal()
+## Vertical offset to help avoid z-fighting, negative values will sink the terrain underneath the road
+@export var offset:float = -0.5
+## Additional flattening to do beyond the edge of the road in meters
+## Note: Not yet implemented
```

Comment:
`edge_margin` and `edge_falloff` are implemented now, right? I see them used down belo